### PR TITLE
Weij/automation test app

### PR DESCRIPTION
**Pull request by weijjia**
Adding automation test app.

### CODE REVIEW

File: automationtestapp/proguard-rules.pro [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
```
Comment (bgavrilMS) [inline automationtestapp/proguard-rules.pro]: This file seems to have only comments - is it needed?

File: automationtestapp/src/test/java/com/microsoft/aad/automation/testapp/ExampleUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.microsoft.aad.automation.testapp;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * To work on unit tests, switch the Test Artifact in the Build Variants view.
+ */
+public class ExampleUnitTest {
```
Comment (bgavrilMS) [inline automationtestapp/src/test/java/com/microsoft/aad/automation/testapp/ExampleUnitTest.java]: I don'\t think we need the sample test

**Comment by bgavrilMS** [comment]
LGTM, although note that the Travis builds are failing

**Comment by kpanwar** [comment]
@weijjia - changes look good but please make sure you address @bgavrilMS's comments

**Comment by bgavrilMS** [comment]
@weijjia - sorry to be pushy about this, but the PR is getting a bit stale and the longer we wait the more difficult it will be to merge. 

Note that the automated tests are using your app for a few days now and it seems to work fine!

File: automationtestapp/src/main/res/layout/activity_result.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/done"
+        android:background="@color/buttonColor"
+        android:id="@+id/resultDone"
+        android:layout_gravity="center_horizontal"/>
+    
+    
+    <TextView
+        android:id="@+id/adalLogs"
```
Comment (unpluggedk) [inline automationtestapp/src/main/res/layout/activity_result.xml]: let's change this to resultLogs as discussed

**Pull request merged**